ztools/fox/fox.go

```diff
@@ -0,0 +1,112 @@
+package fox
+
+import (
+	"encoding/hex"
+	"errors"
+	"net"
+	"strconv"
+	"strings"
+)
+
+const (
+	ORIGINAL_QUERY = "666f7820612031202d3120666f782068656c6c6f0a7b0a" +
```

Comment:
I'm not a huge fan of just putting in a query as binary, but it's not the worst thing.